templates/eap-xp4-basic-s2i.json

```diff
@@ -0,0 +1,545 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
```

Comment:
actually, there is a change I forgot to ask you.

Could you please a namespaced apiVersion to avoid a warning

```
"apiVersion": "image.openshift.io/v1"
```

(similar to https://issues.redhat.com/browse/JBEAP-23082)